### PR TITLE
Add a POC for Open edX - Fonzie shared database

### PR DESCRIPTION
**Pull request by jmaupetit**
**DO NOT MERGE**

This work is a proof-of-concept of making Fonzie a standalone application consumming Open edX data. Here is how we proceeded:

* add a fake Open edX django project running in docker containers (see docker-compose.yml); this is the project that bundles a students app with the Student model that will be exposed by Fonzie
* configure fonzie as a project using the same PostgreSQL database as the Open edX fake project (see test_settings.py)
* implement a serializer and an API view (in fonzie app) for the Student model

And... tada! We were able to expose Open edX data with Fonzie, without having to integrate fonzie in an Open edX instance and without adding Open edX django applications in our `INSTALLED_APPS`.

**Known constrain**: models MUST define an explicit `app_label`

That said, this work stays a POC, and does not take into account Open edX platform black magic and complexity.

The next step is to use the fun-platform dockerized stack and expose Open edX student.UserProfile model as a real world example.

Which me good luck!

### CODE REVIEW

**Comment by jmaupetit** [comment]
I'm closing this PR as it is not intended to be merged but rather a showcase for this POC.

File: fonzie/urls/__init__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+from . import status as status_urls
+from . import students as students_urls
+
+app_name = FonzieConfig.name
+
+API_PREFIX = r'^v(?P<version>[0-9]+\.[0-9]+)'
+
+urlpatterns = [
+    url(r'{}/schema/'.format(API_PREFIX), include(schema_urls, namespace='schema')),
+    url(r'{}/status/'.format(API_PREFIX), include(status_urls, namespace='status')),
```
Comment (sampaccoud) [inline fonzie/urls/__init__.py:20]: Not sure API version in the url is best practice?

File: test_settings.py [hunk 2]
```diff
@@ -16,15 +16,29 @@ def root(*args):
     """
     return join(abspath(dirname(__file__)), *args)
 
+DEBUG = True
+
+ALLOWED_HOSTS = ['testserver', 'localhost', '127.0.0.1']
+
+# DATABASES = {
+#     'default': {
+#         'ENGINE': 'django.db.backends.sqlite3',
```
Comment (sampaccoud) [inline test_settings.py:25]: Dead code ?

File: fonzie/urls/__init__.py (hunk 1 above)
Comment (sampaccoud) [reply fonzie/urls/__init__.py:20]: DRF supports versioning and has a AcceptHeaderVersioning class which looks like a best practice.

Comment (jmaupetit) [reply fonzie/urls/__init__.py:20]: Yup, but unfortunately, for now, `drf_openapi` only supports `URLPathVersioning`. Other versioning scheme are on the project roadmap (see the [project documentation](https://github.com/limdauto/drf_openapi#4-constraints)).

File: test_settings.py (hunk 2 above)
Comment (jmaupetit) [reply test_settings.py:25]: Yes, sorry for that. This branch is not intended to be merged. I have pushed it to show you the code!

**Pull request closed without merging**